client_test.go

```diff
@@ -900,6 +900,89 @@ func TestCreate(t *testing.T) {
 	})
 }
 
+func TestCreateMany(t *testing.T) {
+	ctx := context.Background()
+	client := getClient(t)
+
+	expectedIDs := []map[string]string{
+		{"_id": "1"},
+		{"_id": "2"},
+	}
+
+	resourcesToCreate := []TestResource{
+		{
+			Field:    "v-1",
+			IntField: 1,
+			Nested: NestedResource{
+				Field: "something",
+			},
+		},
+		{
+			Field:    "v-2",
+			IntField: 2,
+			Nested: NestedResource{
+				Field: "something2",
+			},
+		},
+	}
+
+	expectedBody, err := json.Marshal(resourcesToCreate)
+	require.NoError(t, err)
+
+	t.Run("creates resource", func(t *testing.T) {
+		testhelper.NewGockScope(t, baseURL, http.MethodPost, "").
```

Comment:
Here should be "bulk"? Also in other tests. Why do not break the test?

Reply:
I suspect its still matching the prefix :thinking:

Reply:
Turns out it passes because `/resource-path/bulk` matches `$`

Reply:
I think we should enforce a full path match in https://github.com/mia-platform/go-crud-service-client/blob/main/testhelper/gock.go#L98 with `^%s$`.

Reply:
Yes, it is probably the best. We should address it. Can you open an issue?

Reply:
https://github.com/mia-platform/go-crud-service-client/issues/28